luigi/task.py

```diff
@@ -504,6 +506,14 @@ def on_success(self):
         Default behavior is to send an None value"""
         pass
 
+    def set_status_message(self, message):
+        """
+        Sets the status message of the task to message, i.e., invokes _status_message_callback if it
+        is a callable. This propagates the message down to the scheduler.
```

Comment:
In the docs here, can you add a reference the ` _Task.set_status_message:` section? I think there are a few example in this file you can copy paste.